src/components/checkbox/checkbox.scss

```diff
@@ -0,0 +1,401 @@
+@import "default-theme";
+@import "theme-functions";
+@import "variables";
+
+/** The width/height of the checkbox element. */
+$md-checkbox-size: 18px !default;
+/** The width of the line used to draw the checkmark / mixedmark. */
+$md-checkbox-mark-stroke-size: 2/15 * $md-checkbox-size !default;
+/** The width of the checkbox border shown when the checkbox is unchecked. */
+$md-checkbox-border-width: 2px;
+/** The color of the checkbox border. */
+$md-checkbox-border-color: if($md-is-dark-theme, rgba(white, 0.7), rgba(black, 0.54)) !default;
```

Comment:
Could this border color be defined in terms of the theme background / foreground color, or is it unique to the checkbox?